Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg

```diff
@@ -0,0 +1,71 @@
+PART
+{
+// --- general parameters ---
+name = kOSMachineRad
+module = Part
+author = Peter Goddard
+
+// --- asset parameters ---
+mesh = model/model.mu
+scale = 1
+rescaleFactor = 0.99999999999
+iconCenter = 0, 0, 0
+
+// --- node definitions ---
+node_attach = 0.0, 0.0, 0.0, 1, 0, 0
+
+// --- Tech tree ---
+TechRequired = flightControl
+
+// --- editor parameters ---
+cost = 2200
+entryCost = 4200
+category = Control
+subcategory = 0
+title = CompoMax Radial Tubeless
+manufacturer = Squalid-State Devices Inc.
+description = Would you trust life and limb to a mindless autopilot, powered by untested software you hastily wrote yourself? Spacefaring kerbals would!
+
+// attachment rules: stack, srfAttach, allowStack, allowSrfAttach, allowCollision
+attachRules = 0,1,0,0,1
+
+// --- standard part parameters ---
+mass = 0.00001
+dragModelType = default
+maximum_drag = 0.0
+minimum_drag = 0.0
+angularDrag = 0
+crashTolerance = 6
+maxTemp = 3400
+
+
+MODULE
+{
+	name = kOSProcessor
+	diskSpace = 60000

```

Comment:
I would have to defer to you.
I don’t have much interest in the tech tree.
I agree it seems to be a big advantage for unmanned work. Particularly with the combined solar cell.

It makes a heck of a nice sandbox toy.


> On Mar 18, 2015, at 3:28 PM, Chris Woerz <notifications@github.com> wrote:
> 
> In Resources/GameData/kOS/Parts/kOSMachineRad/part.cfg <https://github.com/KSP-KOS/KOS/pull/679#discussion_r26698257>:
> 
> > +attachRules = 0,1,0,0,1
> > +
> > +// --- standard part parameters ---
> > +mass = 0.00001
> > +dragModelType = default
> > +maximum_drag = 0.0
> > +minimum_drag = 0.0
> > +angularDrag = 0
> > +crashTolerance = 6
> > +maxTemp = 3400
> > +
> > +
> > +MODULE
> > +{
> > +	name = kOSProcessor
> > +	diskSpace = 60000
> right now the basic 1.25 part is unlocked in Flight Control and the 0.625 part is unlocked in Precision Engineering. I think this would make most sense in Unmanned Tech or Advanced Unmanned Tech.
> 
> With a solar panel i would say we should not get unlocked before the stock solar panels.
> 
> As far as the disk limit goes, you are right that it is does draw extra power. lower down in the tech tree i am worried that it would draw so much power that it would be unusable.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/KSP-KOS/KOS/pull/679/files#r26698257>.
>